core/src/main/scala/org/apache/spark/scheduler/Task.scala

```diff
@@ -193,7 +192,7 @@ private[spark] object Task {
     dataOut.flush()
     val taskBytes = serializer.serialize(task).array()
```

Comment:
Makes sense.

However, Shixiong rolled back SPARK-12060
Once ByteBufferOutputStream is back, I will update the PR.